src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -1220,38 +1220,286 @@ void Test5()
                 );
         }
 
-        [Fact, WorkItem(25591, "https://github.com/dotnet/roslyn/issues/25591")]
-        public void TupleSubsumptionError()
+        [Fact, WorkItem(25934, "https://github.com/dotnet/roslyn/issues/25934")]
+        public void NamesInPositionalPatterns01()
         {
             var source =
-@"class Program2
+@"class Program
 {
-    public static void Main()
+    static void Main()
+    {
+        switch (a: 1, b: 2)
+        {
+            case (c: 2, d: 3): // error: c and d not defined
+                break;
+        }
+    }
+}
```

Comment:
>element 0 [](start = 78, length = 9)

Are we referring to tuple fields as 0-based elements elsewhere? If not, should we refer to `tuple Item1` here instead?

Reply:
Yes!

---
In reply to: [180581813](https://github.com/dotnet/roslyn/pull/25963#discussion_r180581813) [](ancestors = 180581813)